plugins/c8y_remote_access_plugin/src/config.rs

```diff
@@ -0,0 +1,43 @@
+use miette::Context;
+use miette::IntoDiagnostic;
+use std::net::IpAddr;
+
+pub struct TedgeConfig {
+    pub c8y: TedgeC8yConfig,
+    pub mqtt: TedgeMqttConfig,
+}
+
+pub struct TedgeC8yConfig {
+    pub url: String,
+}
+
+pub struct TedgeMqttConfig {
+    pub port: u16,
+    pub bind_address: IpAddr,
+}
+
+impl TedgeConfig {
+    pub fn read_from_disk() -> miette::Result<Self> {
```

Comment:
Optional: How about implementing a `TryFrom<TEdgeConfig>` for `RemoteAccessPluginConfig` instead of this function?